README.md

```diff
@@ -112,13 +112,25 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
    ***NOTE*** The private key used to connect to the master node is by default (currently hardcoded) at ~/.ssh/id_ibmcloud, so make sure you copy the generated key's private key info into this file.
 
+   Also, add a `addons.yaml` can provide additional add ons in target cluster, for example, download [k8s dashboard](https://raw.githubusercontent.com/kubernetes/dashboard/master/aio/deploy/recommended/kubernetes-dashboard.yaml) and put into `examples/ibmcloud/out/addons.yaml` then add `-a examples/ibmcloud/out/addons.yaml` in `clusterctl` command, after cluster created:
```

Comment:
What about:
s/Also, add a `addons.yaml` can provide additional add ons in target cluster, for example/**Optionally**, add a `addons.yaml` **to** provide additional add ons in target cluster **. For** example/

I think we should tell it is optinally, or else, it will puzzles people why it is necessary.